contentcuration/contentcuration/frontend/channelList/views/ChannelSet/StudioCollectionsTable.vue

```diff
@@ -0,0 +1,345 @@
+<template>
+
+  <KPageContainer class="page-container">
+    <div class="table-header">
+      <div class="header-left">
+        <KButton
+          appearance="basic-link"
+          class="info-link"
+          @click="infoDialog = true"
+        >
+          {{ $tr('aboutChannelSetsLink') }}
+        </KButton>
+
+        <KModal
+          v-if="infoDialog"
+          :title="$tr('aboutChannelSets')"
+          :cancelText="$tr('cancelButtonLabel')"
+          @cancel="infoDialog = false"
+        >
+          <div class="info-content">
+            <p>{{ $tr('channelSetsDescriptionText') }}</p>
+            <p>{{ $tr('channelSetsInstructionsText') }}</p>
+            <p
+              class="disclaimer"
+              :style="{ color: $themeTokens.error }"
+            >
+              {{ $tr('channelSetsDisclaimer') }}
+            </p>
+          </div>
+        </KModal>
+      </div>
+
+      <div class="header-right">
+        <KButton
+          v-if="!loading"
+          appearance="raised-button"
+          primary
+          data-test="add-channelset"
+          :text="$tr('addChannelSetTitle')"
+          @click="newChannelSet"
+        />
+      </div>
+    </div>
+
+    <div class="table-container">
+      <KTable
+        class="collections-table"
+        :stickyColumns="stickyColumns"
+        caption=""
+        :headers="tableHeaders"
+        :rows="tableRows"
+        :dataLoading="loading"
+        sortable
+        disableBuiltinSorting
+        :emptyMessage="$tr('noChannelSetsFound')"
+      >
+        <template #cell="{ content, colIndex }">
+          <!-- Column 0: Collection Name -->
+          <div
+            v-if="colIndex === 0"
+            class="collection-name-cell"
+          >
+            <div class="collection-info">
+              <h3
+                class="collection-title notranslate"
+                dir="auto"
+              >
+                {{ content }}
+              </h3>
+            </div>
+          </div>
+
+          <!-- Column 1: Tokens -->
+          <div
+            v-else-if="colIndex === 1"
+            class="tokens-cell"
+          >
+            <StudioCopyToken
```

Comment:
- Would you make the token field take the full height of a table row, like here in the current version?
<img width="200" height="auto" alt="" src="https://github.com/user-attachments/assets/040579b7-44e1-4805-b955-cb7e2b6b27ec" />
 
 This will optimize how we use space and make rows height smaller.

- If you'd like to, you could also install `kolibri-design-system: https://github.com/MisRob/kolibri-design-system#868060dc647fa556e743715ce4c1a1b77ab453e9` (points to https://github.com/learningequality/kolibri-design-system/pull/1169), and visually hide the input label as described [here](https://deploy-preview-1169--kolibri-design-system.netlify.app/ktextbox/#visually-hidden-label).